tox.ini

```diff
@@ -52,6 +52,8 @@ commands =
 
 [testenv:unit]
 envdir = {toxworkdir}/py3
+deps =
+    backports-datetime-fromisoformat
```

Comment:
This needs to be removed:
- the new library is already included in package deps
- specifying deps here overrides `testenv` deps, meaning removes all useful deps